clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -17,10 +17,11 @@ package clusterdeployer
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/golang/glog"
 	"io/ioutil"
```

Comment:
nit: io/util should probably be grouped with the standard library above.

Reply:
Moved.